website/content/en/status/report-2023-04-2023-06/caldera.adoc

```diff
@@ -1,21 +1,21 @@
 === MITRE Caldera on FreeBSD
 
 Links: +
-link:https://www.caldera.com/[MITRE Caldera] URL: link:https://www.caldera.com/[] +
-link:http://www.redcanary.com[Red Canary] URL: link:http://www.redcanary.com/[] +
+link:https://caldera.mitre.org/[MITRE Caldera] URL: link:https://caldera.mitre.org/[] +
+link:https://www.redcanary.com/[Red Canary] URL: link:https://www.redcanary.com/[]
 
 Contact: José Alonso Cárdenas Márquez <acm@FreeBSD.org>
 
 MITRE Caldera is a cybersecurity platform designed to easily automate adversary emulation, assist manual red teams, and automate incident response.
 
-It is built on the MITRE ATT&CK framework and is an active research project at MITRE.
+It is built on the MITRE ATT&CK(R) framework and is an active research project at MITRE.
 
 MITRE Caldera (package:security/caldera[]) was added to the ports tree in April 2023.
 This port includes support for the link:https://github.com/redcanaryco/atomic-red-team[Atomic Red Team Project] used by the link:https://github.com/mitre/atomic[MITRE Caldera atomic plugin].
 
 The main goal of this work is enhancing visibility of FreeBSD as a useful platform for information security or cybersecurity.
 
-Additionally, you can test a MITRE Caldera infrastructure easily using link:https://github.com/alonsobsd/caldera-makejail[] or link:https://github.com/AppJail-makejails/caldera[] from link:https://github.com/DtxdF/AppJail[AppJail].
+Additionally, you can test MITRE Caldera infrastructure easily using link:https://github.com/alonsobsd/caldera-makejail[] or link:https://github.com/AppJail-makejails/caldera[] from link:https://github.com/DtxdF/AppJail[AppJail].
```

Comment:
Reading this line alongside the other report from José Alonso Cárdenas Márquez, I was probably wrong to suggest this removal of 'a' …